conflictlog/loggerimpl.go

```diff
@@ -235,7 +250,7 @@ func (l *loggerImpl) worker() {
 
 	for {
 		select {
-		case <-l.shutdownCh:
+		case <-l.shutdownWorkerCh:
```

Comment:
When `shutdownWorkerCh` is used, should there be a "clean up" procedure in this cast statement and go through all the `logReqCh`, return error back to each `req.ackCh`?